src/Serilog.Enrichers.Thread/Enrichers/ThreadNameEnricher.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2013-2019 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using System.Threading;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Serilog.Enrichers {
+#if NET45 || NETSTANDARD2_0
```

Comment:
Typically instead of using target frameworks for conditional compilation, we've been using `<DefineProperties>` in the CSPROJ file to define a specific constant, like `THREAD_NAME`, to indicate that the feature is present. This makes it easier to correctly add and change targeted frameworks in future versions. See e.g.: https://github.com/serilog/serilog-sinks-file/blob/dev/src/Serilog.Sinks.File/Serilog.Sinks.File.csproj#L43 and https://github.com/serilog/serilog-sinks-file/blob/dev/src/Serilog.Sinks.File/Sinks/File/SharedFileSink.AtomicAppend.cs#L15

Reply:
Fixed with https://github.com/serilog/serilog-enrichers-thread/pull/11/commits/55cfd64d281e73cef12601831c38669d8d18a3f2

That's nice. I learned something new :-)